txp/Pos/Txp/Settings/Global.hs

```diff
@@ -37,7 +37,6 @@ type TxpCommonMode ctx m =
 type TxpGlobalVerifyMode ctx m =
     ( TxpCommonMode ctx m
     , MonadError ToilVerFailure m
-    , MonadSlots m  -- TODO: I don't like it (@gromak)
```

Comment:
Did you test whether `WITH_EXPLORER` build still works after this change? I suspect it doesn't.